mongo/check.py

```diff
@@ -624,6 +626,18 @@ def _parse_uri(self, server, sanitize_username=False):
 
         return username, password, db_name, nodelist, clean_server_name, auth_source
 
+    def _collect_indexes_stats(self, instance, db, tags):
+        for coll_name in instance.get('collections', []):
+            try:
+                for stats in db[coll_name].aggregate([{"$indexStats": {}}], cursor={}):
+                    idx_tags = tags + [
+                        "name:{0}".format(stats.get('name', 'unknown')),
+                        "collection:{0}".format(coll_name),
+                    ]
+                    self.gauge('mongodb.collection.indexes.accesses.ops', int(stats.get('accesses', {}).get('ops', 0)), idx_tags)
```

Comment:
Would it be worthwhile to also add the "since" metric, to give this data a reference point?

Reply:
Since the agent is reporting every 20s I don't think the `since`  metric is very useful. User could use the `rate` filter to see when the metric stopped increasing. Plus, I don't see any use case in which the time when mongo gathered the metrics is useful.